### PR TITLE
Milestone 6: Portfolio accessibility

### PR DESCRIPTION
**Pull request by Miliyonayalew**
In this milestone @Miliyonayalew  and @taldr27 
We have checked all points on @Miliyonayalew  portfolio website :
- Understood the use of accessibility and why it's important.
- Page title - changed the content title accordingly [Change](https://github.com/Miliyonayalew/Portfolio/pull/11/files#r958889998)
- Image text alternatives** - we added descriptive `alt=""` attribute.  [Change](https://github.com/Miliyonayalew/Portfolio/pull/11/files#r958903388)
- Text headings - changed the page in order to have a hierarchical structure [Change](https://github.com/Miliyonayalew/Portfolio/pull/11/files#r958895341)
- Color contrast - no issues found
- Resize - no issues found
- Interaction - no issues found.
- Moving content - no issues found because we don't have animation.
- Multimedia - no issues found because we don't have media files.
- The basic structure of the page. - no issues found.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Portfolio</title>
+    <title>Miliyon-Ayalew | Portfolio</title>
```
Comment (Miliyonayalew) [inline index.html:7]: Added descriptive title to our project in order to implement the best practice of first review web accessibility.

File: index.html [hunk 2]
```diff
@@ -57,16 +57,16 @@ <h2 class="recent-work-projects-text">My Recent Work</h2>
                             <h3 class="project-title">Multi-Post Stories Gain+Glory</h3>
                             <ul class="tags">
                                 <li class="tag">
-                                    <h6 class="tag-text">Ruby on rails</h6>
+                                    <span class="tag-text">Ruby on rails</span>
```
Comment (Miliyonayalew) [inline index.html:60]: Added `span` instead of `h6` for better hierarchy structure.

File: index.html [hunk 3]
```diff
@@ -214,22 +214,22 @@ <h2 class="about-me-title">About me</h2>
                 <div class="about-me-cards">
                     <div class="about-img-placeholder">   
                         <div class="icons">
-                            <img class="img-icon" src="Assests/Images/icon-languages.svg" alt="language">
+                            <img class="img-icon" src="Assests/Images/icon-languages.svg" alt="language-icon">
```
Comment (taldr27) [inline index.html:217]: We added equivalent user experience `alt=""` attribute to our images.

**Review by victorokoroji — APPROVED** [review]
Hi @Miliyonayalew , 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- Accessibility checks okay ✅ 
- Linters are working properly ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**